docs/content/docs/glossary.md

```diff
@@ -46,3 +46,8 @@ Client code responsible for connecting to the Fluid service.
 
 Fluid's API surface makes use of URLs, for example in the `Loader`'s `resolve()` method and `Container`'s `request()`
 method.  The URL resolver is used to interpret these URLs for use with the Fluid service.
+
+## Aqueduct
+
+The Fluid Framework team's first attempt to reduce boilerplate code and create a more convenient wrapper around the core
+framework.
```

Comment:
```suggestion
The Fluid Framework team's first attempt to reduce boilerplate code and create a more convenient wrapper around the core
framework.
```
The `@fluidframework/aqueduct` package is a library for building Fluid objects and Fluid containers within the Fluid
Framework. Its goal is to provide a thin base layer over the existing Fluid Framework interfaces that allows developers
to get started quickly. [Learn more.](https://fluidframework.com/apis/aqueduct/)

Reply:
The cuggestion didn't work for some reason... I pulled this description from the package readme.